programs/mango-v3-reimbursement/src/instructions/create_reimbursement_account.rs

```diff
@@ -30,3 +31,29 @@ pub fn handle_create_reimbursement_account(
 ) -> Result<()> {
     Ok(())
 }
+
+// non-anchor style helper to make it easier for integrators
+pub fn create_reimbursement_account_instruction(
```

Comment:
nit: actually, adding this crate as a dependency to another non anchor project, would pull in anchor anyway no? Then why not just use anchor cpi directly?

I have no issues merging this

Reply:
Yeah, I assumed it'd be helpful in that it allows you to not worry about learning anchor-isms. You just get a simple function to make an instruction.